src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.MoveFile.cs

```diff
@@ -0,0 +1,238 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using Xunit;
+
+public partial class FileSystemWatcher_4000_Tests
+{
+    #region WindowsTests
+
+    [Theory]
+    [InlineData(WatcherChangeTypes.Changed, "changed", false)]
```

Comment:
I modified the tests to be thread safe but they're still being run sequentially. The FSW before my changes had 53 tests and ran in ~2.9 seconds. After my changes there are 69 tests that take ~8.4 seconds to run.

Modifying the tests to not a bunch of facts instead of theories will only take a few minutes to do since the method bodies just delegate to helper methods. Hopefully xunit will be fine with that.